### PR TITLE
Enable IDE2xxx experimental rules

### LINKED ISSUE #3892: Consider enabling RS0100 on this repo
> Do we want to apply this rule on this repo?

Perhaps, but I like the idea of doing so as a separate PR.

_Originally posted by @sharwell in https://github.com/dotnet/roslyn-analyzers/pull/3888_

### PR DESCRIPTION
**Pull request by Youssef1313**
Fixes #3892

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -242,6 +242,26 @@ dotnet_diagnostic.IDE0130.severity = silent
 # IDE0150: Null check can be clarified
 dotnet_diagnostic.IDE0150.severity = silent
 
+# IDE2000: Allow multiple blank lines
+dotnet_style_allow_multiple_blank_lines_experimental = false
+dotnet_diagnostic.IDE2000.severity = warning
```
Comment (mavasani) [inline .editorconfig:247]: Don’t these rules have category Style? If so, line 185 should already escalate these to warning severity?

**Review by mavasani — APPROVED** [review]
LGTM

I think the dotnet_diagnostic entries added in the PR are unnecessary and can be removed.

File: .editorconfig (hunk 1 above)
Comment (Youssef1313) [reply .editorconfig:247]: Yeah indeed they are not needed. I even had some of these entries as "silent" by mistake. There are more violations to fix.

**Comment by Youssef1313** [comment]
Fix all in Solution for IDE2003 invoked from C# source didn't fix any VB violations. So I needed to run the FixAll twice (once for each language).

Is that expected?

**Comment by mavasani** [comment]
> Fix all in Solution for IDE2003 invoked from C# source didn't fix any VB violations. So I needed to run the FixAll twice (once for each language).
> 
> Is that expected?

Yes that is expected

**Review by mavasani — APPROVED** [review]

**Pull request merged**